IoTBlockly/IoTBlocklyHelper/Gpio.cs

```diff
@@ -57,7 +57,7 @@ public void SetGPIOPin(int pinNumber, int value)
             }
         }
 
-        private GpioPin GetPin(int pinNumber)
+        private static GpioPin GetPin(int pinNumber)
```

Comment:
shouldn't the usage of GetPin switch from <instance>.GetPin to <class>.GetPin?  I don't see a corresponding change?